### ISSUE TITLE
Pluto versions of notebooks?

### ISSUE THREAD

**Issue by metanoid** [SciML/SciMLTutorials.jl#420]
Pluto.jl reached version 0.15 recently.
Would pluto-based versions of these notebooks be a worthwhile addition?

**Comment by ChrisRackauckas** [comment]
Dropping notebooks entirely is best because we want to make the `.md` simple and clean to include into other docs.

**Comment by isaacsas** [comment]
This probably makes more sense to ask about at [Weave.jl](https://github.com/JunoLab/Weave.jl). If they added support to convert Weave .jmd notebooks to Pluto notebooks it would be trivial to add such support here (modulo that the jmd's would need to be written in a style that is consistent with how Pluto notebooks work). 

I do think Pluto notebooks are a nice format for educational tutorials so it would be great if we did get support for such conversions and hosting Pluto versions.

**Comment by ChrisRackauckas** [comment]
Yeah, I think the right way to do the notebooks is lazy generation, since we don't upload the expanded ones anyways. So we could bring back the old `load_notebooks()` function which just opens folds and converts the .jmds to notebooks in a local folder and opens there.
